<ui version="4.0" >
 <class>Chat</class>
 <widget class="QDialog" name="Chat" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>聊天室</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTextBrowser" name="textBrowser" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_to" >
       <property name="text" >
        <string>对</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_whom" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>所有人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_say" >
       <property name="text" >
        <string>说：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
